### DISCUSSION TITLE
Next.js Dev Server Crashes on Reload in Mobile View

### DISCUSSION THREAD

**Discussion by W-Schkulov** [neuefische/questions#862] [Web] [answered]
I’m encountering an issue where my Next.js development server crashes every time I reload the page in mobile view (either on an actual device or in the browser’s responsive design mode). The problem does not occur in desktop view, but consistently crashes when switching to a mobile viewport.

**Comment by eneben** [comment]
Hi, 👋 

I just cloned your project and tried it. The server doesn't crash on me (no matter which view). Does it affect all of you in the project or just you? Only in a certain branch or in `main`?

I would try to clear the browser cache and maybe turn off browser extensions, maybe an extension is causing problems? That would be a good idea if the problem only occurs for you. Then of course restart the server (`npm run dev`). Otherwise, check again to see if you can find any information in the console (also in the Network tab).

Another idea would be if it occurs for all of you and you have certain media queries that are only for mobile devices, whether any errors are triggered?

Good luck. :)

**Comment by F-Kirchhoff** [comment, marked as answer]
Hey Wladimir,
as discussed previously, the problem was your a network setting in your dev tools. The speed throttling was set to "offline". Therefore the browser simulated how the website would behave if the user had no internet connection. By disabling the throttling we solved the issue.

**Reply by W-Schkulov** [reply]
Thank you Felix :)
